bauhaus/workflows/secondary/datasetOps.py

```diff
@@ -13,8 +14,11 @@ def movieName(filename):
 
     base = op.basename(filename)
     fields = base.split(".")
-    assert len(fields) in (3, 4) and fields[-1] == "xml"
-    movieName = fields[0]
+    assert len(fields) > 2 and fields[-1] == "xml"
+    if len(fields) > 3 and re.match(r"chunk[0-9]+", fields[-3]):
+        movieName = ".".join(fields[:-3])
+    else:
+        movieName = ".".join(fields[:-2])
```

Comment:
I'm going to merge for the moment this to unblock Alicia but I may come back to this later with some different thoughts, and I'll adjust the comment.